functions/graphql/schema/flightRecord.js

```diff
@@ -80,6 +85,48 @@ module.exports.resolver = {
     flightRecords: (rootQuery, { userId, centerId, simulatorId }, context) =>
       FlightRecord.getFlightRecords(userId, centerId, simulatorId)
   },
+  Profile: {
+    flightHours: async (profile, args, context) => {
+      let flightHours = await FlightRecord.getFlightRecordsByUser(profile.userId)
```

Comment:
This could definitely be improved by performing it concurrently using Promise.all. I'll merge it and we can improve it later.